core/main/Cargo.toml

```diff
@@ -50,6 +50,12 @@ env-file-reader = "0.2.0"
 sd-notify = { version = "0.4.1", optional = true }
 exitcode = "1.1.2"
 rand = "0.8"
+url.workspace = true
+futures-util = { version = "0.3.28", default-features = false, features = ["sink", "std"] }
```

Comment:
I see `futures-util` imported but not actually used anywhere, am I missing something?

Reply:
Thunder broker uses the sink and std capabilities for doing Websocket operations so it is internally used by the tokio tungstenite.

Reply:
`use futures_util::{SinkExt, StreamExt};`

Reply:
Yes but I didn't see `SinkExt` or `StreamExt` used, maybe I missed that.